system/lib/pthread/library_pthread.c

```diff
@@ -179,12 +178,11 @@ static void _do_call(em_queued_call* q) {
       // disabled. This makes it necessary for us to disable LSan here, so that it does not detect
       // pthread's internal allocations as leaks.
       __lsan_disable();
+#endif
       q->returnValue.i =
-        emscripten_builtin_pthread_create(q->args[0].vp, q->args[1].vp, q->args[2].vp, q->args[3].vp);
+        __pthread_create_js(q->args[0].vp, q->args[1].vp, q->args[2].vp, q->args[3].vp);
+#ifdef HAS_SANITIZER
```

Comment:
Note to myself: these `__lsan_{disable,enable}();` calls can probably be removed if we also move the pthread stack allocation to native code.
https://github.com/emscripten-core/emscripten/blob/b44d69e865102973cb92bb55bda1af8fe3093579/src/library_pthread.js#L703-L705

Reply:
Interesting.. I didn't realize that is what they were for.  I'll add a comment, thanks!